src/mono/wasm/debugger/DebuggerTestSuite/GetPropertiesTests.cs

```diff
@@ -427,8 +437,12 @@ private static void AssertHasOnlyExpectedProperties(string[] expected_names, IEn
                 {"BaseBase_MemberForOverride",                  TString("DerivedClass#BaseBase_MemberForOverride")},
 
                 // inherited public
-                {"Base_AutoStringProperty",         TString("base#Base_AutoStringProperty")},
-                {"LastName",                        TGetter("LastName")}
+                {"Base_AutoStringProperty",                                     TString("base#Base_AutoStringProperty")},
+                {"LastName",                                                    TGetter("LastName")},
+                {"BaseBase_MemberForOverride (BaseClass)",                      TGetter("BaseBase_MemberForOverride (BaseClass)")},
+                {"Base_AutoStringPropertyForOverrideWithField (BaseClass)",     TObject("string", is_null: true)},
+                {"Base_GetterForOverrideWithField (BaseClass)",                 TGetter("Base_GetterForOverrideWithField (BaseClass)")},
+                {"BaseBase_MemberForOverride (BaseBaseClass)",                  TString(null)}
```

Comment:
Rename `BaseBase_MemberForOverride` to `BaseBase_MemberForHiding` since it seems to be used mainly for `new`-hiding, and it would make it obvious in the tests, that it should be different.